packages/ETLoad.package/ETLObjectHeader.class/instance/hash.st

```diff
@@ -0,0 +1,7 @@
+comparing
+hash
+	^ self sizeInWords hash
```

Comment:
This hash implementation probably works like this for now, but in our last project it caused sever performance issues.